### ISSUE TITLE
Combo box component doesn't work properly with NVDA Screen Reader

### ISSUE THREAD

**Issue by Oleksandr-Sirenko** [brianvoe/slim-select#623]
Using keyboard navigation, I try to open the combobox component. By default, the search input field is hidden. The problem is that the screen reader still reads the search input field even though it is hidden. I think this is due to tabIndex=0.
Using the Up/Down arrows on the keyboard, I can navigate between the options. However, the screen reader pronounces "Blank" each time I highlight an option, instead of reading the value of the option.

This bug can be reproduced on the homepage:
https://slimselectjs.com/

1. Install NVDA for Windows:
https://www.nvaccess.org/download/;
2. Run NVDA;
3. Using Tab navigation, navigate to the combobox component;
4. Open the combobox using the Enter key;
5. Try to navigate between the options in the dropdown list;
6. NVDA, on each highlight change, pronounces "Blank".
